cart/src/main/java/com/yas/cart/repository/CartItemV2Repository.java

```diff
@@ -26,6 +26,7 @@ public interface CartItemV2Repository extends JpaRepository<CartItemV2, CartItem
     @Query("SELECT c FROM CartItemV2 c WHERE c.customerId = :customerId AND c.productId = :productId")
     Optional<CartItemV2> findByCustomerIdAndProductId(String customerId, Long productId);
 
+    @Query("SELECT c FROM CartItemV2 c WHERE c.customerId = :customerId ORDER BY c.createdOn")
```

Comment:
Seems you want to include `ORDER BY` to the query, let's use JPA Query Method instead of `@Query`, it's also support `ORDER BY`.

Reply:
Thank you, I agree. I'll switch to using JPA Query Method with ORDER BY. Will update accordingly.